code/drasil-example/Drasil/Projectile/IMods.hs

```diff
@@ -33,17 +33,18 @@ import Drasil.Projectile.Unitals (flightDur, landPos, launAngle, launSpeed,
 iMods :: [InstanceModel]
 iMods = [timeIM, landPosIM, offsetIM, messageIM]
 
+iMods0 :: [InstanceModel]
+iMods0 = [landPosIM, offsetIM, messageIM]
```

Comment:
We need to make sure that these are the only IMs for which we use `relToQD` on as well, in the Body.hs